### ISSUE TITLE
change image size for training

### ISSUE THREAD

**Issue by Phyrokar** [Deci-AI/super-gradients#1144]
### 🐛 Describe the bug

Hallo, 

I tried to change Image size during training with 

```
val_dataset_params.input_dim = (720, 1280)
train_dataset_params.input_dim = (720, 1280)
```

but it seems like it doesnt actually influence the image size. Can someone tell me how I can make this work?

### Versions

super-gradients 3.1.1
